doc/design/diagrams/iceoryx_software_layers_v3_0_0.puml

```diff
@@ -0,0 +1,38 @@
+@startuml
```

Comment:
Why do we version by filename and not git?

Reply:
I thought it's nicer to keep at least the `.puml` of the former release, so users can compare what has changed. Storing the diagrams of the 10 last releases wouldn't make sense.